### DISCUSSION TITLE
publish/unpublish/delete flows - what should we do for the UX and how to deal with it in the various replication modes

### DISCUSSION THREAD

**Discussion by reusr1** [peregrine-cms/enhancements#58] [Ideas]
unpublishing of content (pages, assets, tags) is a bit inconsistent in peregrine at the moment - we should find a common UX flow for unpublish that also works with the different replication modes.

https://github.com/peregrine-cms/enhancements/issues/56
https://github.com/peregrine-cms/enhancements/issues/54
https://github.com/peregrine-cms/enhancements/issues/48

**Comment by ehdatheadwire** [comment]
Short term solutions was to block unpublish or delete of items that are in use. We can circle back around to this when we have time to dig into improving the overall UX flow. @ehdatheadwire to double check tickets referenced above to see if they can be closed.
